libs/barista-components/filter-field/src/filter-field-range/filter-field-range-trigger.ts

```diff
@@ -289,20 +280,13 @@ export class DtFilterFieldRangeTrigger implements OnDestroy {
 
   /** Returns the overlay position. */
   private _getOverlayPosition(): PositionStrategy {
-    /**
-     * @breaking-change Switch to only use the DtFlexibleConnectedPositionStrategy
-     * with 7.0.0
-     */
-    const originalPositionStrategy =
-      this._viewportRuler && this._platform && this._overlayContainer
-        ? new DtFlexibleConnectedPositionStrategy(
-            this._elementRef,
-            this._viewportRuler,
-            this._document,
-            this._platform,
-            this._overlayContainer,
-          )
-        : this._overlay.position().flexibleConnectedTo(this._elementRef);
+    const originalPositionStrategy = new DtFlexibleConnectedPositionStrategy(
+      this._elementRef,
+      this._viewportRuler,
+      this._document,
+      this._platform,
+      this._overlayContainer,
+    );
     this._positionStrategy = originalPositionStrategy
```

Comment:
The variable `originalPositionStrategy` was only necessary because we chained things to both versions afterwards. I guess this is no longer necessary. 

```suggestion
    this._positionStrategy = new DtFlexibleConnectedPositionStrategy(
      this._elementRef,
      this._viewportRuler,
      this._document,
      this._platform,
      this._overlayContainer,
    );
```